server.go

```diff
@@ -138,8 +140,12 @@ func NewServer(options ...func(*Server)) *Server {
 		option(s)
 	}
 
+	if s.Server.Addr == "" {
+		WithAddr("localhost:9999")(s)
+	}
+
 	s.OpenApiSpec.Servers = append(s.OpenApiSpec.Servers, &openapi3.Server{
-		URL:         "http://" + s.Addr,
+		URL:         "http://" + s.Server.Addr,
```

Comment:
Shouldn't it be https:// when isTLS is true?

Reply:
right, i get protocol from proto() method